book/modules/appendix3.tex

```diff
@@ -66,18 +67,22 @@
 
 	Alternatively, row reducing and keeping track of the elementary matrices for each step, we see
 	\[
-		\mat{1 & -6 \\ 0 & 1}\mat{1 & 0 \\ 0 & -\frac{1}{5}}\mat{1 & 0 \\ -2 & 1}M=\mat{1&0\\0&1},
+		\underbrace{\mat{1 & -6 \\ 0 & 1}}_{\textstyle E_3}
+		\underbrace{\mat{1 & 0 \\ 0 & -\frac{1}{5}}}_{\textstyle E_2}
+		\underbrace{\mat{1 & 0 \\ -2 & 1}}_{\textstyle E_1} M = \mat{1&0\\0&1},
 	\]
 	and so
 	\[
-		M=\mat{1 & 0 \\ 2 & 1}\mat{1 & 0 \\ 0 & -5}\mat{1 & 6 \\ 0 & 1}=E_1E_2E_3.
+		M = \mat{1 & 0 \\ 2 & 1}
+		    \mat{1 & 0 \\ 0 & -5}
+		    \mat{1 & 6 \\ 0 & 1} = E_1^{-1} E_2^{-1} E_3^{-1}.
```

Comment:
Changed the elementary matrices to their inverses because that's how we did it in a similar example from module 14.